client/components/signup-form/index.jsx

```diff
@@ -308,12 +309,35 @@ class SignupForm extends Component {
 		} );
 	};
 
+	getNoticeMessageWithLogin( notice ) {
+		const link = login( {
+			isNative: config.isEnabled( 'login/native-login-links' ),
+			redirectTo: this.props.redirectToAfterLoginUrl
+		} );
```

Comment:
Super minor but we could declare this variable inside the `if` clause since it's the only place in this method where it's used.